examples/PTCDA_Hartree_dz2/example_ptcda_hartree.py

```diff
@@ -0,0 +1,27 @@
+import os
+import subprocess
+from pathlib import Path
+
+from ppafm.cli.generateElFF import main as generate_elff
+from ppafm.cli.generateLJFF import main as generate_ljff
+from ppafm.cli.plot_results import main as plot_results
+from ppafm.cli.relaxed_scan import main as relaxed_scan
+
+
+def example_ptcda_hartree():
+    script_location = Path(__file__).absolute().parent
+
+    # Change directory to the location of this script
+    os.chdir(script_location)
+
+    subprocess.run(["wget", "--no-check-certificate", "https://www.dropbox.com/s/18eg89l89npll8x/LOCPOT.xsf.zip"])
+    subprocess.run(["unzip", "LOCPOT.xsf.zip"])
+
+    generate_ljff(["--input", "LOCPOT.xsf"])
+    generate_elff(["--input", "LOCPOT.xsf", "--tip", "dz2"])
+    relaxed_scan(["--klat", "0.5", "--charge", "-0.10"])
+    plot_results(["--klat", "0.5", "--charge", "-0.10", "--Amplitude", "0.5", "--df"])
+
+
+if __name__ == "__main__":
+    example_ptcda_singe()
```

Comment:
Wrong function, `example_ptcda_singe()` is being run twice now. Also `singe` -> `single`.

Reply:
Pytest was looking for functions starting with `example_` so both `example_ptcda_hartree` and `example_ptcda_singe` were run.

Anyway, thanks for the careful review, I fixed those things in c451e7c91c1f1e65ec0fe9649e45080cc7b5b875